x/mongo/driver/topology/topology_test.go

```diff
@@ -160,9 +163,8 @@ func TestServerSelection(t *testing.T) {
 			t.Errorf("Timed out while trying to retrieve selected servers")
 		}
 
-		if err != context.Canceled {
-			t.Errorf("Incorrect error received. got %v; want %v", err, context.Canceled)
-		}
+		want := ServerSelectionError{Wrapped: context.Canceled, Desc: desc}
+		assert.Equal(t, err, want, "Incorrect error received. got %v; want %v", err, want)
```

Comment:
I'd like this test to assert that the `Unwrap` function works as expected. You can do that using the `errors.Is` function (https://blog.golang.org/go1.13-errors). An example of this in our existing tests: https://github.com/mongodb/mongo-go-driver/blob/master/mongo/integration/errors_test.go#L36. Note that you'll need to add a build flag to the top of this file so it only runs tests if compiled with Go 1.13 or higher: https://github.com/mongodb/mongo-go-driver/blob/master/mongo/integration/errors_test.go#L7.